WordPress/gutenberg
# Font Library: Set filename using the font face data to the font assets

## What?
Use the font face data to generate the font file asset name.

## Why?
Fix: https://github.com/WordPress/gutenberg/issues/58411

## How?
Generating a filename based on the fontface definition.

## Testing Instructions
Install a font face from a collection. Verify that the filename of the assets is related to the font face installed.
